web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -274,23 +276,48 @@ export function ProjectCard({
                     )}
                   </div>
                 )}
-                <div className="flex gap-10">
-                  {containedButton && onContainedButtonClick && (
-                    <ContainedButton
-                      size="small"
-                      startIcon={containedButton.startIcon}
-                      disabled={containedButton.disabled}
-                      sx={{ width: '100%' }}
-                      className={containedButton.className}
-                      onClick={onContainedButtonClick}
-                    >
-                      {containedButton.text}
-                    </ContainedButton>
-                  )}
+                <div className="grid gap-10 grid-cols-2">
```

Comment:
this breaks ProjectCard's with only one button, see home page or /projects page for instance: 

<img width="1283" alt="image" src="https://github.com/user-attachments/assets/89a80fe9-9125-42e7-987d-5b557aa7ad04">

Reply:
fixed